### DISCUSSION TITLE
Glade upgrade to 3.40?

### DISCUSSION THREAD

**Discussion by Blady-Com** [jralls/gtk-osx-build#74] [Ideas]
Hello,

In current moduleset, Glade version is 3.38.2.
Glade 3.40.0 was released this summer.
What about updating?

Thanks, Pascal.

**Comment by jralls** [comment]
Pascal, I generally do updates once a year around February unless something is broken--zlib being a recent example--but I'm always open to MRs on gitlab or PRs here if you're motivated to build and test an updated module.

**Comment by jralls** [comment]
BTW, from the release notes for 3.40:
"This is the last stable release.

"Glade is not being actively developed or maintained anymore. If you are interested in becoming a maintainer please contact us on gitlab."

**Comment by Blady-Com** [comment]
Updated in commit https://github.com/jralls/gtk-osx-build/pull/75.
